recipes/xapian/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "xapian" %}
+{% set version = "1.4.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://oligarchy.co.uk/xapian/{{ version }}/xapian-bindings-{{ version }}.tar.xz
+  sha256: 4519751376dc5b59893b812495e6004fd80eb4a10970829aede71a35264b4e6a
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - xapian-core={{ version }}
+    - zlib
```

Comment:
Here (and in the other recipe), I would expect only the compiler in `reqs/build` and the xapian-core and zlib to be in `reqs/host`. It's worth standardizing if you didn't have a particular need for all in the build section.

Also, how tightly bound (version-wise) is xapian to xapian-core? If it's as you indicate, `xapian-core {{ version }}` or  `xapian-core {{ version }}*` may be a better syntax. If the two packages need to be bound down to the build number, there's ways to address that.